src/shogun/base/SGObject.cpp

```diff
@@ -817,23 +795,19 @@ class CSGObject::ParameterObserverList
 };
 
 void CSGObject::register_observable_param(
-    const std::string& name, const SG_OBS_VALUE_TYPE type,
-    const std::string& description)
+    const std::string& name, const std::string& description)
 {
-	param_obs_list->register_param(name, type, description);
+	param_obs_list->register_param(name, description);
 }
 
-void CSGObject::list_observable_parameters()
+std::vector<std::string> CSGObject::list_observable_parameters()
```

Comment:
This guy here returns directly a vector of strings (with the name of the observable parameters), but I am not sure if this what you intended @karlnapf.

Reply:
sorry. I was distracted by the name. What about observable_names ... which is inline with recent changes for normal parameters. Minor though